.github/pull_request_template.md

```diff
@@ -14,8 +21,6 @@
 
 <!-- please IGNORE the below if you aren't a CIL team member
 
-blame GH for this text: https://github.com/orgs/community/discussions/81319
```

Comment:
why to remove this link? It may be useful to check in the future?

Reply:
I find it a pain to delete every time I make a PR

Reply:
Could open an issue instead which has the link in?

Reply:
#2203